build.sbt

```diff
@@ -219,9 +219,14 @@ lazy val pureconfig = myCrossProject("pureconfig")
   .dependsOn(core % "compile->compile;test->test")
   .settings(
     libraryDependencies ++= macroParadise(Test).value ++ Seq(
-      "com.github.pureconfig" %% "pureconfig-core" % pureconfigVersion,
-      "com.github.pureconfig" %% "pureconfig-generic" % pureconfigVersion % Test
-    )
+      "com.github.pureconfig" %% "pureconfig-core" % pureconfigVersion
+    ) ++ (CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12 | 13)) =>
+        Seq(
+          "com.github.pureconfig" %% "pureconfig-generic" % pureconfigVersion % Test
+        )
+      case _ => Seq.empty
+    })
```

Comment:
Would it be much work to rewrite the Scala 2 tests without `pureconfig-generic`? It would be nice if we could get rid of `pureconfig-generic` and this complication in the build.

Reply:
Sure thing, will do now.


> ```
> [error]    |                        value of is not a member of object java.util.Map
> ```

Fixed. My bad, didin't test with old Java.

As for generic, I'll be happy to fix soon